.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+  push:
+    branches:
+      - main
+      - master
+
+env:
+  CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+jobs:
+  run:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install hatch
+      - name: Test
+        env:
+          VAULT_TOKEN: ${{ secrets.VAULT_TOKEN }}
+          YOUTRACK_TOKEN: ${{ secrets.YOUTRACK_TOKEN }}
```

Comment:
Where are these secrets coming from again? I can't see them in https://github.com/vimc/vault-dev/settings/secrets/actions

Reply:
Oh they are not set in the test https://github.com/vimc/vault-dev/actions/runs/6507016485/job/17673653219?pr=2#step:5:12 are they needed for the testing then?

Reply:
aha, I copied this over from a different setup, these can be removed!